minecraft_servers/verart/manifest.json

```diff
@@ -0,0 +1,64 @@
+{
+    "server_name": "verart",
+    "nice_name": "Verart",
+    "direct_ip": "verart.org",
+    "server_wildcards": [
+        "%.verart.org"
```

Comment:
Doesn't look like a real wildcard:
![image](https://github.com/user-attachments/assets/d11a78f8-32bd-4f22-bb65-98a770735a2b)

Reply:
Hey, can you check now if this has been resolved?

Reply:
Looks good 👍